src/dropdown.test.basics.multiple.ts

```diff
@@ -52,54 +53,28 @@ it('sets `value` to that of its selected options', async () => {
       <glide-core-dropdown-option
         label="Three"
         value="three"
-        selected
       ></glide-core-dropdown-option>
     </glide-core-dropdown>`,
   );
 
-  expect(host.value).to.deep.equal(['two', 'three']);
-});
-
-it('has selected option labels when options are selected', async () => {
```

Comment:
Deleted this test after moving it and the below test's assertions into the test above.